### PR TITLE
160-feature/animal module

### LINKED ISSUE #160: Add animal module
Add a new module with following functionalities

- bear
- bird
- cat
- cetacean
- cow
- crocodilia
- dog
- fish
- horse
- insect
- lion
- rabbit
- rodent
- snake
- type

### PR DESCRIPTION
**Pull request by Globostofo**
Took data from [faker-js](https://github.com/faker-js/faker)

Added animals:
- bears
- birds
- cats
- cetaceans
- cows
- crocodilia
- dogs
- fishes
- horses
- insects
- lions
- rabbits
- rodents
- snakes
- types (generates a random string from the previous animal)

Closes #160

### CODE REVIEW

File: include/faker-cxx/Animal.h [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <string>
+
+namespace faker
+{
+class Animal
+{
+public:
+    static std::string bear();
```
Comment (cieslarmichal) [inline include/faker-cxx/Animal.h:10]: Please add documentation to each method like in other classes.

**Review by cieslarmichal — CHANGES_REQUESTED** [review]
Good job, left one comment :)

**Comment by Globostofo** [comment]
Oops, I forgot documentation.

It's good now, I added it :)

**Comment by cieslarmichal** [comment]
I see the branch is out of date, could you update it please?

**Review by cieslarmichal — APPROVED** [review]

**Pull request merged**